predict.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+import json
+from gino import Gino
+from tornado.web import RequestHandler, MissingArgumentError
+CREDENTIALS_FILE = "credentials.json"
```

Comment:
It's better to just use a credentials.py file and import that instead of using JSON, it's cleaner and you don't have to import the JSON module and parse a file, which is relatively resource intensive.

credentials.py:
```python
username = 'abc'
password = 'def'
```

Importing in another file:
```python
import secret
username = secret.username
password = secret.password
```

Reply:
Hmmm true, will fix that tomorrow